src/WrapperApp/components/Results/EstimatorsSelect/EstimatorsSelect.tsx

```diff
@@ -0,0 +1,126 @@
+import { Card, CardContent, Tab, Tabs } from '@mui/material';
+import { SyntheticEvent, useCallback, useEffect, useState } from 'react';
+
+import { FullSimulationData, useShSimulation } from '../../../../services/ShSimulatorService';
+
+interface EstimatorsTabProps {
+	tabsValue: number;
+	setTabsValue: (value: number) => void;
+	simulation: FullSimulationData;
+	setResultsSimulationData: React.Dispatch<React.SetStateAction<FullSimulationData | undefined>>;
+	estimatorsTabData: string[];
+	isFlukaExample: boolean;
+}
+
+const EstimatorsSelect = ({
+	tabsValue,
+	setTabsValue,
+	simulation,
+	setResultsSimulationData,
+	estimatorsTabData,
+	isFlukaExample
+}: EstimatorsTabProps) => {
+	const [controller] = useState(new AbortController());
+
+	const { getJobResult } = useShSimulation();
+
+	const handleEstimatorTabChange = useCallback(
+		async (id: number) => {
+			const currentEstimatorData = simulation.estimators;
+			const estimatorExists =
+				currentEstimatorData.some(estimator => estimator.name === estimatorsTabData[id]) ||
+				isFlukaExample;
```

Comment:
why do you need to make Fluka-specific logic ? the same should apply to all type of examples: Fluka, SHIELD-HIT12A and any other future simulator

setting the tab name should be done in consistent name for all simulators